www/stos02/sem_web/styles.css

```diff
@@ -313,7 +319,7 @@
   ul{
         margin-left:10px;
     }
-  .main_head h1{
+  #name{
       font:bold 10px Tahoma;
       top:10px;
```

Comment:
kaskadovost:
nemely by opakovat vlastnosti, ktere maji stejnou hodnotu, tj. v media query uvest jen to, co se zmenilo, pripadne nove vlastnosti